index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/8f283d01ad.js" crossorigin="anonymous"></script>
+    <title>Daron Wiafe (Software Developer)</title>
+</head>
+<body>
+    <div id="wrapper">
+        <nav class="portfolio_nav">
+            <img src="daron.jpeg" alt="author of the website">
```

Comment:
- Kindly your profile picture does not belong to the original Figma design.
- It should contain your name and a link to the home page.